climada/hazard/tc_tracks.py

```diff
@@ -1450,6 +1451,35 @@ def _one_interp_data(track, time_step_h, land_geom=None):
             track_land_params(track_int, land_geom)
         return track_int
 
+def _xr_to_netcdf_multi(path, ds_dict, encoding=None):
+    """Write multiple xarray Datasets to separate groups in a single NetCDF4 file
+
+    Contrary to xarray's `to_netcdf` functionality this only supports the "NETCDF4" format and the
+    "netcdf4" engine since the groups feature has been introduced by NetCDF version 4.
+
+    Parameters
+    ----------
+    path : str or Path
+        Path of the target NetCDF file.
+    ds_dict : dict whose keys are group names and values are xr.Dataset
+        Each xr.Dataset in the dict is stored in the group identified by its key in the dict.
+        Note that an empty string ("") is a valid group name and refers to the root group.
+    encoding : dict whose keys are group names and values are dict, optional
+        For each dataset/group, one dict that is compliant with the format of the `encoding`
+        keyword parameter in `xr.Dataset.to_netcdf`. Default: None
+    """
+    path = str(pathlib.Path(path).expanduser().absolute())
+    store = NetCDF4DataStore.open(path, "w", "NETCDF4", None)
+    try:
```

Comment:
You should handle the exception in some way at least.

Reply:
No, see my comment above.